app/views/shared/_composables.html.erb

```diff
@@ -0,0 +1,27 @@
+<%- data.each do |section| -%>
```

Comment:
Yep! That's what I'm thinking. We'll end up generating all the composable stuff in to the project. Especially since this is the front-end view! Wouldn't be very nice having this stuck in Koi.